pkg/mdns/routes.go

```diff
@@ -82,7 +82,7 @@ func (c *MicroShiftmDNSController) waitForRouterAPI(dc dynamic.Interface, router
 	if err != nil {
 		// This is a soft error, the watcher down the line will keep waiting, but will emit
 		// less nice errors
-		klog.Errorf("waiting for the route.openshift.io/v1 API to come up", err)
+		klog.Errorf("waiting for the route.openshift.io/v1 API to come up: %s", err)
```

Comment:
It's an error, so should we say something like the following?
Failed waiting for...

Reply:
Done.